org.moneymanagerex.MMEX.yml

```diff
@@ -0,0 +1,35 @@
+app-id: org.moneymanagerex.MMEX
+runtime: org.gnome.Platform
+runtime-version: '44'
+sdk: org.gnome.Sdk
+command: mmex
+finish-args:
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --device=dri
+  - --filesystem=home
+  - --share=ipc
+modules:
+  - name: wxWidgets
+    config-opts:
+       - --disable-shared
+       - --enable-cxx11
+       - --with-cxx=11
+       - --enable-webview
```

Comment:
check the build log. it likely doesn't find webview probably it want 4.0 while the runtime has 4.1.